### ISSUE TITLE
elasticache nuke error

### ISSUE THREAD

**Issue by denis256** [gruntwork-io/cloud-nuke#782]
Noticed that elasticache removal fails with error:
```
Nuking batch of 1 elasticache resource(s) in eu-central-1 [19/931]  2% | 6sERRO[2024-11-11T10:09:41Z] runtime error: invalid memory address or nil pointer dereference  binary=cloud-nuke version=e4e9493b52bd500015b49e2b3ed88a93fc3811fe
```

**Comment by denis256** [comment]
[log-1.txt](https://github.com/user-attachments/files/17702641/log-1.txt)

**Comment by wakeful** [comment]
Hey, I was able to reproduce the issue by creating a **single-node cluster**. I addressed the problem in #783.
